techniques/css/C42.html

```diff
@@ -90,24 +73,26 @@ <h2>Tests</h2>
 				<h3>Procedure</h3>
 
 				<p>For each target that cannot be enlarged by a mechanism, is not inline, and is not covered by the essential exception:</p>
```

Comment:
Wondering where the "that cannot be enlarged by a mechanism" comes from. It's not in the normative text of the SC. I see there's a proposed (but not actually created?) sufficient technique "Providing a mechanism to change the size of the target independent of magnification" mentioned in https://www.w3.org/WAI/WCAG21/Understanding/target-size.html ... but we should make techniques independent of each other. So I'd say remove this bit.

Reply:
done.